client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -47,8 +49,25 @@ registerBlockType( name, settings );
 
 const onboardingBlock = createBlock( name, {} );
 
-registerCoreBlocks();
-const templateBlock = createBlock( 'core/paragraph', { content: 'Template Selection' } );
+const DesignSelector = () => {
+	const templates = useSelect( select =>
+		select( 'automattic/verticals/templates' ).getTemplates( 'p13v1' )
+	);
+	return <PageTemplateModal templates={ templates } />;
+};
+
+// Makeshift block so we can drop the modal into the block editor. Might want to change that later.
+registerBlockType( 'automattic/page-templates', {
+	title: 'Page Templates',
```

Comment:
should we translate the title?

Reply:
Sorry, this is far from being ready for review. I think GitHub has you folks listed as code owners and thus automatically added as reviewers :man_facepalming:

Reply:
:-) happens 🤷‍♂